chapters/annotations.tex

```diff
@@ -475,6 +477,18 @@ \section{Annotations for Simulation Experiments}\label{annotations-for-simulatio
 
 The \lstinline!experiment! annotation defines the default start time (\lstinline!StartTime!) in {[}s{]}, the default stop time (\lstinline!StopTime!) in {[}s{]}, the suitable time resolution for the result grid (\lstinline!Interval!) in {[}s{]}, and the default relative integration tolerance (\lstinline!Tolerance!) for simulation experiments to be carried out with the model or block at hand.  If \lstinline!StartTime! is not specified it is assumed to be \lstinline!0.0!.
 
+\subsection{Annotation for Test Cases}\label{annotation-for-test-cases}
+\begin{lstlisting}[language=grammar]
```

Comment:
Another way to structure _Annotations for Simulations_ would be to use two `annotationdefinition` environments instead of two subsections.  This way, the new annotation would automatically end up in the index.

Reply:
Resolving with the same resolution as the comment above.